### PR TITLE
feat(core): Rename several packages to remove redundant "ceramic-" prefix

### PR DESCRIPTION
**Pull request by stbrody**

### CODE REVIEW

**Comment by stbrody** [comment]
So, I redid this patch from scratch, and for some reason this time I had no issues, never hit a circular dependency with identity-wallet, and never had to actually do anything fancy outside of js-ceramic to make it work.  Not sure what I did different this time, or if something else changed in js-ceramic in the meantime, but either way it seems to work now so I'll take it!

**Comment by stbrody** [comment]
FYI @simonovic86 @PaulLeCam @zachferland @valmack - just a heads up that this is coming down the pipe and will likely cause fallout many of our other repos.

**Comment by PaulLeCam** [comment]
Could we use a "next breaking changes release" branch instead of develop maybe to make multiple breaking changes at once and schedule a specific timeline for this release and the necessary changes to other packages please?
As we have the SkyDB hackathon starting tomorrow I think we should avoid pushing breaking changes until it's over.

**Review by oed — APPROVED** [review]
Looks good.
And valid concern @PaulLeCam, could we still merge this to develop but wait with making a release until we merged a lot more breaking changes?
Basically keep using current master and merge hotfixes there and release if needed?

**Review by simonovic86 — COMMENTED** [review]
LGTM

**Comment by simonovic86** [comment]
> Could we use a "next breaking changes release" branch instead of develop maybe to make multiple breaking changes at once and schedule a specific timeline for this release and the necessary changes to other packages please?
> As we have the SkyDB hackathon starting tomorrow I think we should avoid pushing breaking changes until it's over.

@PaulLeCam yeah, I wanted to propose a same thing. Separate branch is definitely a safer way to go 👍

**Comment by PaulLeCam** [comment]
> And valid concern @PaulLeCam, could we still merge this to develop but wait with making a release until we merged a lot more breaking changes?
> Basically keep using current master and merge hotfixes there and release if needed?

Sounds good to me but up to @simonovic86, whatever is simpler for you.

File: packages/core/src/__tests__/dispatcher.test.ts
Comment (stbrody) [reply packages/core/src/__tests__/dispatcher.test.ts]: done

**Pull request merged**
